webpack.config.js

```diff
@@ -139,6 +139,12 @@ var config = {
                     },
                 ],
             },
+            {
+                test: /\.m?js/,
```

Comment:
What is this change for? There's a few things I notice that are potentially incorrect here:
1. This can capture plain old `.js` files because the `m` is optional which seems like we wouldn't want to have happen
2. Similarly, it can also capture `.jsx` files because it doesn't check for the end of the string with `$`

Reply:
fixed